Doctrine/Phpcr/StaticContent.php

```diff
@@ -11,14 +11,13 @@
 
 namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr;
 
-use Doctrine\ODM\PHPCR\HierarchyInterface;
 use PHPCR\NodeInterface;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent as ModelStaticContent;
 
 /**
  * PHPCR specific static content.
  */
-class StaticContent extends ModelStaticContent implements HierarchyInterface
```

Comment:
hm. the HierarchyInterface also has getParent / setParent deprecated. ideally we would synchronize the removal there to keep the interface on the content.